src/screens/professorHome/components/subjectsList/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Typography } from '@mui/material';
+import { TCompleteSubject } from '../../../../service/requests/useGetSubject/types';
+import MonitorsList from '../monitorsList';
+import {
+  AccessSubject,
+  Container,
+  FallbackContainer,
+  SubjectHeaderContainer,
+} from './styles';
+import LoadingAnimation from '../../../../components/loadingAnimation';
+
+type Props = {
+  isLoading: boolean;
+  filteredSubjects: TCompleteSubject[];
+};
+
+const SubjectsList = ({ isLoading, filteredSubjects }: Props) => {
+  if (isLoading) {
+    return (
+      <>
+        <FallbackContainer>
+          <LoadingAnimation />
+        </FallbackContainer>
+      </>
```

Comment:
Não precisa dos brackets quando retorna só um componente:

```suggestion
      <FallbackContainer>
        <LoadingAnimation />
      </FallbackContainer>
```

Reply:
resolvido